### DISCUSSION TITLE
About validation rules

### DISCUSSION THREAD

**Discussion by stratboy** [rahulhaque/laravel-filepond#54] [Q&A] [answered]
Hi, what exactly are "file" and "max:5000" for?

```
    'validation_rules' => [
        'required',
        'file',
        'max:5000',
    ],
```

And what validation rules can we use? Are there other rules we can use?

**Comment by rahulhaque** [comment, marked as answer]
@stratboy when you drop a file in the filepond, these are the rules that get validated first before storing it in the temp folder. Without this rule, user can literally upload anything consuming your whole server space. This is done to prevent users from uploading files you don't want in your system.

**Reply by stratboy** [reply]
ok thank you but what exactly check the 'file' and max:5000? That is a 'file' (but could it be something other than a 'file')? That size is max 5000 bytes (that would be really small, that's why I'm wandering what this really is)? Is it maybe kb?

**Reply by rahulhaque** [reply]
It's in kilobytes. See the validation doc - https://laravel.com/docs/10.x/validation

**Reply by stratboy** [reply]
Ah ok it's Laravel, not your package! Perfect, thank you!
